tests/model_serving/maas_billing/multitenancy/aitenant/test_invalid_namespace_placement.py

```
@@ -0,0 +1,67 @@
+    ) -> None:
+        """Verify the admission webhook rejects AITenant CRs outside ai-tenants."""
+        aitenant_name, cr_namespace = aitenant_outside_infra_namespace
+        aitenant_spec = build_aitenant_spec(aitenant_name=aitenant_name)
+        aitenant = aitenant_from_spec(
+            admin_client=admin_client,
+            aitenant_name=aitenant_name,
+            cr_namespace=cr_namespace,
+            aitenant_spec=aitenant_spec,
+            teardown=teardown_resources,
+        )
+        with pytest.raises(DynamicApiError, match="(?i)admission webhook"), aitenant:
```

Ruff (0.15.18)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)